content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -628,3 +628,83 @@ If the :abbr:`PA (Public Administration)` business doesn't reply within 15 days,
+the printer is required.
+
+.. image:: italy/epson-90-keyboard.png
+   :alt: Fiscal printer keyboard
+
+.. image:: italy/epson-90-screen.png
+   :alt: Fiscal printer screen
+
+To set up the fiscal printer using the keyboard, follow these steps:
+
+#. Make sure that the fiscal printer is set to its default state: printer on, start-up cycle
+   complete, and no transaction in progress.
+#. Type `3333`.
+#. Press :guilabel:`Chiave`. The screen will display :guilabel:`Scleta Funzione`.
+#. Type `34`. The screen will display :guilabel:`Web Server`.
+#. Press :guilabel:`Contante` 3 times until the screen displays :guilabel:`Web Server: SSL`.
+#. To turn the value `0` into a `1`, press :guilabel:`X`.
+#. To confirm, press :guilabel:`Contante` 3 times.
+#. Press :guilabel:`Chiave`.
+
+Then, log in to the printer with the device that runs the :doc:`point of sale
+</applications/sales/point_of_sale>` so it recognizes the printer's certificate.
+
+To approve and install the printer's certificate, follow these steps:
+
+#. To access the printer, open a web browser and enter `https://<ip-of-your-printer>` in the address
+   bar. A :guilabel:`Warning: Potential Security Risk Ahead` security message will then appear.
+#. To show the certificate approval options, click :guilabel:`Advanced`.
+#. To validate the certificate, click :guilabel:`Proceed`.
+
+Then, to ensure the :doc:`point of sale </applications/sales/point_of_sale>` is configured with the
+fiscal printer, go to :menuselection:`Point of Sale --> Configuration --> Settings`. In the
+:guilabel:`Connected Devices` section, add the IP address in the :guilabel:`Italian Fiscal Printer
+IP address` and enable :guilabel:`Use HTTPS`.
+
+Configuring Simulation Mode
+---------------------------
+
+It is possible to test the setup of the fiscal printer with Odoo. To do this, configure the fiscal
+printer in Simulation mode with the following steps:
+
+#. Make sure that the fiscal printer is set to its default state: printer on, start-up cycle
+   complete, and no transaction in progress.
+#. Type `3333`.
+#. Press :guilabel:`Chiave`. The screen will display :guilabel:`Scleta Funzione`.
+#. Type `14`. The screen will display :guilabel:`Apprendimento`.
+#. Type `62`. The screen displays :guilabel:`Simulazione`.
+#. To turn the :guilabel:`no` into a :guilabel:`si`, press :guilabel:`X`.
+#. To confirm, press :guilabel:`Contante`.
+#. Press :guilabel:`Chiave`.
+
+To configure the printer to production, follow the above setps once more.
+
+..note::
+   In oder to test the printer configuration, the actual physical unit must be acquired and
+   registered with the authorities.
+
+..warning::
+   As simulation mode does send the data to the authorities, setting simulation should only be done
+   at the very beginning of the configuration of the printer. Once the printer has been sey to
+   production, it should not be set to simulation again.
```

Comment:
As this is an important piece of information to give from the start, I'd prefer to add it immediately after the "Configuring Simulation Mode" heading.
```suggestion
.. warning::
   Since simulation mode transmits data to the authorities, it should only be enabled at the very 
   start of the printer's configuration process. After the printer is switched to production mode,
   it must not be reverted to simulation mode.
```